protocols/1181-laboratory-specialists-inc/README.md

```diff
@@ -0,0 +1,42 @@
+# Minimum Inhibitory Concentrations (MIC) Dilution Prep
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+## Categories
+* Basic Pipetting
+    * Dilution
+
+## Description
+This protocol allows the robot to prepare serial dilutions (15 dilutions) in 2 columns of a 96-deep well plate and then transfer the solutions to a clean 96-well plate. This protocol requires a P10 multi-channel and P300 multi-channel pipettes.
+
+stock_concentration: float=1600,
```

Comment:
You might want to specify these are default values, I think most people wouldn't understand this syntax.